React/Views/RCTModalHostView.h

```diff
@@ -21,6 +21,7 @@
 @interface RCTModalHostView : UIView <RCTInvalidating>
 
 @property (nonatomic, copy) NSString *animationType;
+@property (nonatomic, copy) NSString *presentationStyle;
```

Comment:
Please use `RCTConvert` to map strings to enum values (see `RCTScrollView` for examples).